### ISSUE TITLE
Enhancement: Allow text wrap for Description field in Diagnostics panel

### ISSUE THREAD

**Issue by skylize** [facebook-atom/atom-ide-ui#158]
### Description
I am, in most cases, only interested in the first result shown Diagnostics, and want to be able to find out as much information as possible about that result with additional activity with the mouse.

Current behavior of Diagnostics is to limit output to a single line, which does not work well with long results or smaller than screen-width panel size.

I would _much_ prefer needing to scroll down for additional diagnostic results, over needing to hover to get the full details of the first result.

### Expected Behavior
Long descriptions in Diagnostics will wrap to fit available width, pushing down any additional diagnostic results to make space.

### Actual Behavior
Diagnostics limits description to a single line, requiring hovering mouse for tooltip to see any text that does not fit in the available width.

### Versions
* Atom: 1.24.0-beta3
* Client OS: Windows 10.0.17074.1002
* atom-ide-ui: 0.7.2

**Comment by matthewwithanm** [comment]
Is this not happening if you toggle the "Full Description" toggle in the panel? (the `atom-ide-diagnostics-ui.showDiagnosticTraces` setting?)

**Comment by skylize** [comment]
No, that does not help. The only effect from that setting is that when set to Off it adds some ellipses instead of just truncating at the edge of the bounding box.

![image](https://user-images.githubusercontent.com/844400/35883549-8f121b44-0b55-11e8-9ba8-b1944332b348.png)

**Comment by captbaritone** [comment]
Often line breaks in diagnostic messages are meaningful (Flow results for example). I worry that allowing wrapping would add confusion to the 90% use case in order to support this less common use case.

**Comment by skylize** [comment]
@captbaritone I am not implying this should be a _forced_ change. A toggle would be ideal, a setting that actually does what it claims an acceptable consolation. 

I failed to notice the `Show full diagnostic traces` setting when I first posted this as an "enhancement". This is actually appears to be a bug in `atom-ide-diagnostics-ui.showDiagnosticTraces`, as the output with the setting `on` or `off` can only be distinguished by `on` hard-clipping the hidden text instead of showing a nice ellipses. It is certainly not showing me a "full trace", as demonstrated in the image above.

**Comment by Bartozzz** [comment]
Full description still not working on:
- `atom-ide-ui`: v.0.11.2
- `Atom`: v.1.26.1
- `Electron`: v.1.7.11
- `Chrome`: v.58.0.3029.110
- `Node`: v.7.9.0

### Error in Diagnostics:

<img width="939" src="https://user-images.githubusercontent.com/6741707/39945470-d93d2830-556a-11e8-8512-ba126f6201a8.png">

### Full error:

<img width="502" src="https://user-images.githubusercontent.com/6741707/39945505-00a02d96-556b-11e8-8a8f-82e3b169a785.png">
